### DISCUSSION TITLE
Shopify

### DISCUSSION THREAD

**Discussion by jb-1892** [saadeghi/daisyui#2126] [Q&A] [answered]
Hi,

Just wondering before I delve further into this rabbit hole - Has anyone got DaisyUI working with Shopify? (not hydrogen/react).

I've been trying to get it running for days, Tailwind works fine but nothing from Daisy. I just wanted to ask in case it's simply not compatible?

Happy to provide more details if I find out anyone else has it working.

Thanks 

Jamie

**Comment by saadeghi** [comment]
If Tailwind works, daisyUI should work as well.  
Can you share a sample repo or your config+file structure?

**Comment by jb-1892** [comment]
I know, it's a strange one! Here's the repo, thanks for coming back to me

**Comment by saadeghi** [comment, marked as answer]
In your `tailwind.config.js` file, `plugins` shouldn't be inside `purge`

**Comment by jb-1892** [comment]
Oops! You're quite right. This didn't fix anything though, here's my updated config file (I tried with and without purge):
```
module.exports = {
 
  content: ["./layout/*.liquid", "./templates/*.liquid", "./sections/*.liquid", "./snippets/*.liquid", "./src/js/*.js"],
  plugins: [require("daisyui")]

}
```

**Comment by saadeghi** [comment]
> Oops! You're quite right. This didn't fix anything though, here's my updated config file (I tried with and without purge):
> 
> ```
> module.exports = {
>  
>   content: ["./layout/*.liquid", "./templates/*.liquid", "./sections/*.liquid", "./snippets/*.liquid", "./src/js/*.js"],
>   plugins: [require("daisyui")]
> 
> }
> ```

With this config, if you run your script `npm run build:css`, daisyUI classes will be added to `/assets/application.css.liquid` file. I tried and it worked. I don't see a problem.

**Comment by jb-1892** [comment]
Thank you so much! I really appreciate your help, and more so your time. 

I guess running that script within the "go" script was the issue, I'll look into that. Daisy looks incredible, it's been killing me not being able to get started using it.

Thanks again , you're a legend
